cypress/integration/models/keycloak/users/user.ts

```diff
@@ -0,0 +1,108 @@
+import { UserData } from "../../../types/types";
+import { button, SEC, tdTag, trTag } from "../../../types/constants";
+import {
+    click,
+    clickByText,
+    clickWithin,
+    inputText,
+    performRowAction,
+} from "../../../../utils/utils";
+import * as loginView from "../../../views/login.view";
+const tackleUiUrl = Cypress.env("tackleUrl");
+const adminPassword = Cypress.env("adminPassword");
```

Comment:
Let's rename this to keycloackConsoleUrl ..
otherwise it is confusing

Reply:
Shveta, this is really Tackle URL, not keycloak URL. I'm getting same URL as in all other places. Keycloak URL I'm getting on line 28, please take a look. I won't change the name here.